### DISCUSSION TITLE
7BC89DB4-893A-45A3-9B27-B24C83BD85FD

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#199] [General]
# 7BC89DB4-893A-45A3-9B27-B24C83BD85FD

Week 34, 2024: Jupiter-bound Spacecraft flies by the Earth and the Moon (August 19–20, 2024)

https://celestia.mobi/article?guide=7BC89DB4-893A-45A3-9B27-B24C83BD85FD

**Comment by Chriskyle-star** [comment]
Wowww

**Comment by Complitated** [comment]
Ok
